torch/onnx/_internal/exporter/_dispatching.py

```diff
@@ -241,10 +241,10 @@ def get_matching_overload(
     for overload in overloads:
         assigned_types: dict[str, ir.TypeProtocol] = {}
         fail_reason = ""
-        if not hasattr(overload, "signature"):
+        if overload.signature is None:
```

Comment:
What if overload is an operator? (operator.mul, ...)

Reply:
overload is type OnnxDecompMeta. When it stores an operator, overload.signature is None.